src/modules/people-list/__test__/people-list-item.test.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {render, fireEvent} from '@testing-library/react-native';
+
+import ThemeProvider from '../../theming/theme-provider';
+import defaultTheme from '../../theming/default-theme';
+
+import UserItem from '../people-list-item';
+
+const onPressMock = jest.fn();
+
+describe('UserItem', () => {
+  describe('Rendering', () => {
+    it('should render properly', () => {
+      const wrapper = render(
+        <ThemeProvider theme={defaultTheme}>
+          <UserItem
+            name={'Stephen John'}
+            maritalStatus={'Married'}
+            gender={'M'}
+            churchEntityName={'Corona'}
+            profilePic={'https://www.google.com'}
+            userId={'512345'}
+          />
+        </ThemeProvider>,
+      );
+
+      const headerWrapper = wrapper.queryByTestId('main-wrapper');
+
+      expect(headerWrapper).toBeDefined();
+
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('hides icon when it is null', () => {
+      const wrapper = render(
+        <ThemeProvider theme={defaultTheme}>
+          <UserItem
+            name={'Stephen John'}
+            maritalStatus={'Married'}
+            gender={'M'}
+            churchEntityName={'Corona'}
+            profilePic={null}
+            userId={'512345'}
+          />
+        </ThemeProvider>,
+      );
+
+      const icon = wrapper.queryByTestId('profile-pic');
+
+      expect(icon).toBeNull();
+
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('Clicked user item', () => {
```

Comment:
This test case should in a separate block called "Interactions". Test cases are usually grouped together based on the type of testing being performed. The Rendering block ([see example](https://github.com/saddlebackdev/hc-mobile-app-ui/blob/dev/src/modules/avatar/__tests__/avatar.test.tsx#L12)) houses everything in terms of how the component looks and the Interactions block ([see example](https://github.com/saddlebackdev/hc-mobile-app-ui/blob/dev/src/modules/avatar/__tests__/avatar.test.tsx#L84)) houses everything related to how the component behaves when interacted with it.

Reply:
I have created a separate block called "Interactions".